spec/Umpirsky/PermissionsHandler/ConfigurationSpec.php

```diff
@@ -1,39 +1,41 @@
 <?php
 
-namespace spec\Umpirsky\PermissionsHandler;
+declare(strict_types=1);
+
+namespace Spec\Umpirsky\PermissionsHandler;
 
-use PhpSpec\ObjectBehavior;
-use Composer\Script\Event;
 use Composer\Composer;
 use Composer\Package\PackageInterface;
+use Composer\Script\Event;
+use PhpSpec\ObjectBehavior;
 
 class ConfigurationSpec extends ObjectBehavior
 {
-    function let(Event $event, Composer $composer, PackageInterface $package)
+    public function let(Event $event, Composer $composer, PackageInterface $package)
```

Comment:
We don't put visibility in specs. They just decrease readability.
```suggestion
    function let(Event $event, Composer $composer, PackageInterface $package)
```